game_eggs/steamcmd_servers/american-truck-simulator/egg-ats.json

```diff
@@ -0,0 +1,31 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v2",
+        "update_url": null
+    },
+    "exported_at": "2022-12-10T21:09:36-05:00",
+    "name": "American Truck Simulator Dedicated Server.",
+    "author": "chase@cstmgames.com",
+    "description": "This will set up SteamCMD and all the required files for the newly released American Truck Simulator Dedicated server.  Simply placed the exported server config from your ATS instance to the .local\/share\/American Truck Simulator\/ folder and select start.",
+    "features": null,
+    "docker_images": {
+        "ghcr.io\/parkervcp\/games:source": "ghcr.io\/parkervcp\/games:source"
+    },
+    "file_denylist": [],
+    "startup": "$(pwd)\/bin\/linux_x64\/amtrucks_server",
```

Comment:
this can just be ./bin/linux_x64/amtrucks_server as the entrypoint will force you to be in /home/container